portfolio/src/main/java/com/google/sps/Response.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+/** Sets instance Response of a message object */
+public final class Response {
+
+  private final String firstname;
```

Comment:
typically functions in javascript should be camel case (ie firstName, lastName, etc) please reconsider and update this to fit into the typical style.

Reply:
It's a java file. And I followed example from the walk-through in creating my own.